catalogData/redis32/ha/3x/k8s/sentinel-proxy-deployment.json

```diff
@@ -0,0 +1,45 @@
+{
+  "apiVersion": "apps/v1beta1",
+  "kind": "Deployment",
+  "metadata": {
+    "name": "redis-sentinel-proxy"
+  },
+  "spec": {
+    "replicas": 1,
```

Comment:
yeah. I totally left that in there while I was testing this stuff out on dev/locally